tests/unit/parallel/segmented_algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2017 Ajai V George
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set(tests
+    partitioned_vector_copy
+    partitioned_vector_for_each
+    partitioned_vector_handle_values
+    partitioned_vector_iter
+    partitioned_vector_move
+    partitioned_vector_target
+    partitioned_vector_transform
+    partitioned_vector_transform_reduce
+    partitioned_vector_fill
+    partitioned_vector_inclusive_scan
+    partitioned_vector_exclusive_scan
+    partitioned_vector_transform_scan
+    partitioned_vector_reduce
+   )
+
+set(partitioned_vector_copy_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_for_each_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_for_each_PARAMETERS
+    LOCALITIES 2
+    THREADS_PER_LOCALITY 4)
+set(partitioned_vector_transform_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_transform_PARAMETERS
+    LOCALITIES 2
+    THREADS_PER_LOCALITY 4)
+set(partitioned_vector_handle_values_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_iter_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_move_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_transform_reduce_FLAGS DEPENDENCIES partitioned_vector_component)
+set(partitioned_vector_transform_reduce_PARAMETERS
+    LOCALITIES 2
+    THREADS_PER_LOCALITY 4)
+set(partitioned_vector_reduce_FLAGS DEPENDENCIES partitioned_vector_component)
```

Comment:
Is there a specific reason why `reduce` test is not executed on multiple localities?